expiry/bin/prune.inc.php

```diff
@@ -1,13 +1,12 @@
 <?php
 // XXX Uncomment these lines if you run into memory limit failures
-// ini_set('max_execution_time', 3000);
```

Comment:
revert this change, and all of the subsequent commits, and I will merge. The other issues (adjusting sql queries, adding the link limit) would please be a separate request. There is another way I may go with addressing the sql issue.